pkg/services/udp_proxy_test.go

```diff
@@ -150,3 +153,111 @@ func TestUDPProxyServiceOutbound(t *testing.T) {
 		})
 	}
 }
+
+func TestProcessInboundPacket(t *testing.T) {
+	expectedAddr := netceptor.Addr{}
+	expectedAddr.SetNetwork("tcp")
+	expectedAddr.SetNode("127.0.0.1")
+	expectedAddr.SetService("2222")
+	expectedUDPAddr := &net.UDPAddr{IP: net.ParseIP("127.0.0.1"), Port: 2222}
+	unexpectedUDPAddr := &net.UDPAddr{IP: net.ParseIP("127.0.0.1"), Port: 3333}
+	tests := []struct {
+		name           string
+		expectedAddr   netceptor.Addr
+		calls          func(mockPacketConner *mock_netceptor.MockPacketConner, mockUDPConnInterface *mock_net_interface.MockUDPConnInterface)
```

Comment:
```suggestion
		calls          func()
```
This can be simplified to just `func()` and you can declare these as variables in the test. Example is on lines 99-101

Then the following test cases just have the func() heading makes it easier to read.

Reply:
Yeah... I changed this along with moving this inside the test loop:
```
	ctrl := gomock.NewController(t)
	defer ctrl.Finish()
```
Because I was getting less flakiness this way. You are right and defining the pointers at test scope is the way to go.